docs/readmes/lte/architecture_overview.md

```diff
@@ -29,7 +29,50 @@ Note: The *mme* service will be renamed soon to *accessd*, as its main purpose i
 
 Together, these components help to facilitate and manage data both to and from the user.
 
-![AGW diagram](assets/agw_services.png)
+
+```
+     ┏━━━━━━━━━━━━━━┓           ┏━━━━━━━━━━━━━━┓
+     ┃ Magma Orc8r  ┃───────────┃  Magma Feg   ┃
+     ┗━━━━━━━━━━━━━━┛           ┗━━━━━━━━━━━━━━┛
+             ▲      ▲
+             │      │
+       ┏━━━━━┻━━━━━━┻━━━━━┓
+╔══════╣  control_proxy   ╠═══════════════════════════════════════════Access Gateway (AGW)
+║      ┗━━━━━┳━━━━━━┳━━━━━┛                                                              ║
+║            │      └────────────────────────────────────────────────┐                   ║
+║            └──────────────────────────────────┐                    │                   ║
+║                                               │                    │                   ║
+║ ┏━━━━Magma EPC━━━━┓         ┏━━━━Session━━━━┓ │  ┏━━━━Device━━━━━┓ │┏━━━Optional━━━━┓  ║
```

Comment:
EPC is the whole thing not only access side. Instead simply put Access Control & Management. Can you write (accessd) in parenthesis next to MME?